File: RSPAClient/Core/AtPar.POCOEntities/MT_ATPAR_PHYSICAL_INVENTORY.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_ATPAR_PHYSICAL_INVENTORY
    {
        public string ORG_ID { get; set; }
        public string INV_ITEM_ID { get; set; }
        public string UOM { get; set; }
        public string STOR_LOC { get; set; }
        public string CHARGE_CODE { get; set; }
        public string LOT_ID { get; set; }
        public string SERIAL_ID { get; set; }
        public string DEFAULT_LOC_FLAG { get; set; }
        public Nullable<int> QUANTITY_ON_HAND { get; set; }
        public Nullable<System.DateTime> LAST_UPDATE_DATE { get; set; }
        public string REPLENISHMENT_TYPE { get; set; }
        public Nullable<bool> STATUS { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_ATPAR_PHYSICAL_INVENTORY.ts
export class MT_ATPAR_PHYSICAL_INVENTORY {
    public ORG_ID: string;
    public INV_ITEM_ID: string;
    public UOM: string;
    public STOR_LOC: string;
    public CHARGE_CODE: string;
    public LOT_ID: string;
    public SERIAL_ID: string;
    public DEFAULT_LOC_FLAG: string;
    public QUANTITY_ON_HAND?: number;
    public LAST_UPDATE_DATE?: Date;
    public REPLENISHMENT_TYPE: string;
    public STATUS?: boolean;
}